src/Lykke.Service.Assets/Startup.cs

```diff
@@ -182,15 +206,13 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, IApplica
             {
                 // NOTE: Service not yet receive and process requests here
 
-                var cqrs = _applicationContainer.Resolve<ICqrsEngine>(); // bootstrap
```

Comment:
Is it ok to remove this line?

Reply:
This var wasn't used in the block of code. After removal, we have no compilation\runtime conflicts. I assume it's safe.